expression/builtin_miscellaneous_test.go

```diff
@@ -221,3 +221,31 @@ func (s *testEvaluatorSuite) TestInet6AtoN(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(r.IsNull(), IsTrue)
 }
+
+func (s *testEvaluatorSuite) TestIsIPv4Mapped(c *C) {
+	tests := []struct {
+		ip     []byte
+		expect interface{}
+	}{
```

Comment:
Nope, we can not do that, cause `net.ParseIP`  parse  a mapped IPv6 address into its origin IPv4 address. It not works as expected. So we need directly use `[]byte` to do that.